notes.md

```diff
@@ -27,3 +27,8 @@ POST: not idempotent (some effect)
 - Add new endpoint? Get all drafts for user?
 - Continue to save drafts every 2 seconds? Or use separate button? (UX)
 - Make PR
+
+- Add button to save draft
```

Comment:
would love to see more details about how you planned to implement this feature. E.g. you decided to list all the drafts in a list next to invoice editor together with "delete" button - that is a UX decision that we could discuss before implementing the feature.